### ISSUE TITLE
Replace istanbul module

### ISSUE THREAD

**Issue by gavinr** [Esri/esri-leaflet-geocoder#247]
when running `npm install`, I get this message:

```
npm WARN deprecated istanbul@0.4.5: This module is no longer maintained, try this instead:
npm WARN deprecated   npm i nyc
npm WARN deprecated Visit https://istanbul.js.org/integrations for other alternatives.
```

**Comment by gavinr** [comment]
I've done a bit more research here. 

- It looks like this message is coming from [`isparta`](https://github.com/douglasduteil/isparta), which we're using [here in `karma.conf.js`](https://github.com/Esri/esri-leaflet-geocoder/blob/581d68977e71478d30b7de0871bfd4aef2a5bfaf/karma.conf.js#L64-L69), which we fixed a similar issue in `esri-leaflet` [here](https://github.com/Esri/esri-leaflet/commit/415fa78f7db1d86e1ccf095eb66659e9211bcbcc).

**Comment by gavinr** [comment]
This was released in [v2.3.3](https://github.com/Esri/esri-leaflet-geocoder/releases/tag/v2.3.3)
